pkg/build/api/v1/types.go

```diff
@@ -402,6 +402,11 @@ type SourceBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// disableImplicitBuild prevents source-to-image from performing a Docker build operation
```

Comment:
An end user wants this so they are not blocked from doing this if their admin trusts them but is not willing to open it up to the entire cluster.

And an end user wants this if they want to protect themselves from having to inspect every s2i builder image to see if it contains rogue ONBUILD commands that amount to trojan horse exploits.